Makefile

```diff
@@ -363,8 +373,8 @@ deploy-test-%:
 	oc new-project $(TEST_NAMESPACE) || true
 	oc adm policy add-scc-to-user anyuid -z default -n $(TEST_NAMESPACE)
 	oc adm policy add-scc-to-user privileged -z default -n $(TEST_NAMESPACE)
-	oc apply -n $(TEST_NAMESPACE) -f deploy/bookinfo/session_role.yaml
-	oc apply -n $(TEST_NAMESPACE) -f deploy/bookinfo/session_rolebinding.yaml
+	oc apply -n $(TEST_NAMESPACE) -f deploy/examples/bookinfo/session_role.yaml
+	oc apply -n $(TEST_NAMESPACE) -f deploy/examples/bookinfo/session_rolebinding.yaml
```

Comment:
Maybe remove "bookinfo" level or rename it while we're at it?

Reply:
Yeah, that `bookinfo` thing is irrelevant at this point.

Reply:
db1b00f